dataset/dictionary/weapons.json5

```
@@ -243,7 +243,7 @@
     en: "Wolf-Fang",
     ja: "狼牙",
     zhCN: "狼牙",
-    pronunciationJa: "ろうが", // TODO Need Check
+    pronunciationJa: "ろうが",
```

Please revert this if there are no official sources.

```suggestion
    pronunciationJa: "ろうが", // TODO Need Check
```